lib/rucio/tests/test_rule.py

```diff
@@ -153,6 +156,74 @@ def setup_class(request, vo, root_account, jdoe_account, rse_factory):
 @pytest.mark.usefixtures('setup_class')
 class TestCore:
 
+    @pytest.mark.parametrize("caches_mock", [{"caches_to_mock": [
+        'rucio.core.rse_expression_parser.REGION'
+    ]}], indirect=True)
+    @pytest.mark.parametrize(
+        "delete_rse", [True, False]
+    )
+    def test_move_rule_invalid_source_replica_expression(
+        self,
+        caches_mock,
+        delete_rse: bool,
+        did_factory: "TemporaryDidFactory",
+        rse_factory: "TemporaryRSEFactory",
+        root_account: "InternalAccount"
+    ):
+        """
+        REPLICATION RULE (CORE): Move a rule using an invalid source replica expression.
+
+        Parametrised so that there are two different ways for the source replica
+        expression (SRE) to become invalid:
```

Comment:
> Also changed mentions of SRE to the typed out Source Replica Expression.

Did you?

Reply:
I did so in the commit message, but forgot to do so in the source code itself.

Reply:
Changed reference to SREs within the docstring, but did not change the variable name `SRE` in the test code.